src/components/Accounts.js

```diff
@@ -60,27 +53,29 @@ const Account = ({ user }) => {
                     buttons={[['outline-danger', () => { deleteUser(); navigate('/login'); }, 'Delete'],
                               ['success-outline', () => {}, 'Cancel']]}>
                   </CustomAlert>
-                  <h1>Hello {user && user.username}</h1>
+                  <h1>Hello {getuname}</h1>
                   <div>Email: {user && user.email}</div>
-                  <Link to='/account/allergies'>Allergies</Link>
-                  <br></br>
-                  <Link to='/account/filters'>Restaurant Preferences</Link>
-                  <br></br>
-                  <Link to='/history'>Restaurant History</Link>
-                  <br></br>
-                  <button>
-                    <Link to='/change-password'>Change Password</Link>
-                  </button>
-                  <br></br>
-                  <button onClick={sOut}>Sign Out</button>
-                  <br></br>
-                  <button onClick={() => { setShow(true) }}>Delete Account</button>
-                  <Form onSubmit={handleSubmit}>
-                    <Button className="w-40 mt-10" type="submit">
-                      Home
+
+                  <div id = 'btn'>
```

Comment:
ids should be unique. Refactor into class

Reply:
with a different name, bootstrap has classes with these names that we use in various places around the app so name them something else

Reply:
how about you make me